judgments/models.py

```diff
@@ -14,13 +14,17 @@ class Meta:
         }
 
     metadata_name = xmlmodels.XPathTextField(
-        "//akn:FRBRname/@value", ignore_extra_nodes=True
+        "/akn:akomaNtoso/akn:judgment/akn:meta/akn:identification/akn:FRBRWork/"
+        + "akn:FRBRname/@value",
+        ignore_extra_nodes=True,
     )
     neutral_citation = xmlmodels.XPathTextField(
         "//akn:proprietary/uk:cite", ignore_extra_nodes=True
     )
     date = xmlmodels.XPathTextField(
-        "//akn:FRBRdate[@name='judgment']/@date", ignore_extra_nodes=True
+        "/akn:akomaNtoso/akn:judgment/akn:meta/akn:identification/akn:FRBRWork/"
+        + "akn:FRBRdate/@date",
```

Comment:
This will impact the published dates in the ATOM feeds too, I strongly suspect, which could do with documenting. Feel free to make a card for me to do that on Wednesday.